packages/web-react/src/components/Heading/README.md

```diff
@@ -3,15 +3,16 @@
 The Heading component provides helper classes to render headings.
 
 ```jsx
-<Heading elementType="h1" size="large" />
+<Heading elementType="h1" size="large" emphasis="bold" />
 ```
 
 ## API
 
-| Name          | Type                                        | Default  | Required | Description      |
-| ------------- | ------------------------------------------- | -------- | -------- | ---------------- |
-| `elementType` | `React.Element`                             | `div`    | ✕        | HTML tag         |
-| `size`        | [Size Extended dictionary][dictionary-size] | `medium` | ✕        | Size of the text |
+| Name          | Type                                        | Default  | Required | Description          |
+| ------------- | ------------------------------------------- | -------- | -------- | -------------------- |
+| `elementType` | `React.Element`                             | `div`    | ✕        | HTML tag             |
+| `emphasis`    | [`italic` \| `bold` \| `semibold`]          | —        | ✕        | Emphasis of the text |
```

Comment:
```suggestion
| `emphasis`    | \[`italic` \| `bold` \| `semibold`]          | —        | ✕        | Emphasis of the text |
```

Please, also escape the first char in this case, so the markdown check will not throw an error. Because Markdown thinks this is a reference.